sdk/eventhub/Azure.Messaging.EventHubs/src/Primitives/PartitionReceiverOptions.cs

```diff
@@ -0,0 +1,173 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+using Azure.Core;
+using Azure.Messaging.EventHubs.Core;
+
+namespace Azure.Messaging.EventHubs.Primitives
+{
+    /// <summary>
+    ///   The set of options that can be specified when creating a <see cref="PartitionReceiver" />
+    ///   to configure its behavior.
+    /// </summary>
+    ///
+    internal class PartitionReceiverOptions
+    {
+        /// <summary>The set of options to use for configuring the connection to the Event Hubs service.</summary>
+        private EventHubConnectionOptions _connectionOptions = new EventHubConnectionOptions();
+
+        /// <summary>The set of options to govern retry behavior and try timeouts.</summary>
+        private EventHubsRetryOptions _retryOptions = new EventHubsRetryOptions();
+
+        /// <summary>The amount of time to wait for messages when reading.</summary>
+        private TimeSpan _defaultMaximumReceiveWaitTime = TimeSpan.FromMinutes(1);
+
+        /// <summary>The prefetch count to use for the partition receiver.</summary>
+        private int? _prefetchCount = null;
```

Comment:
Based on our offline conversation, you're correct in that we should align this to the version from the Event Processor